Signal/src/ViewControllers/MediaMessageView.swift

```diff
@@ -36,15 +42,17 @@ class MediaMessageView: UIView, OWSAudioAttachmentPlayerDelegate {
 
     @available(*, unavailable, message:"use attachment: constructor instead.")
     required init?(coder aDecoder: NSCoder) {
+        self.mode = .large
```

Comment:
TIOLI: You can use `fatalError("uninimplemented")` and stop worrying about stubbing bogus data in methods we don't use.

Reply:
Nice!